components/script/dom/htmlelement.rs

```diff
@@ -1135,7 +1135,7 @@ impl VirtualMethods for HTMLElement {
                         evtarget.remove_event_listener(
                             event_name.into(),
                             evtarget.get_event_handler_common(event_name, can_gc),
-                            EventListenerOptions { capture: false },
+                            None,
```

Comment:
Can you check why the test linked in https://github.com/servo/servo/pull/38734#issuecomment-3194447459 is still failing with this patch? I would expect it to be fixed, which was the original motivation for the issue you linked

Reply:
Seems like inline event listeners `onclick` are not removed in this check, only additive ones `addEventListener("click", ...)`. Will see how I can add an equivalence check for these listeners.

Reply:
I think we should be using set_inline_event_listener instead of remove_event_listener here? It doesn't make sense to run removeEventListener on inline event listeners (`onclick`, etc).